tests/unit/set_instance_tags/test_get_synapse_tags.py

```diff
@@ -1,30 +1,38 @@
 import unittest
+import boto3
```

Comment:
When the code for the two app.py's is consolidated, the repeated tests will go away.